### ISSUE TITLE
what DUA to pick in case of there not being any data?

### ISSUE THREAD

**Issue by robertoostenveld** [Donders-Institute/rdm-wiki#22]
I just wrote "In case the data is not to be shared, the written request by the researcher and the approval of the center director will be uploaded to the Data Sharing Collection." 

That is in accordance with the discussion we had with the directors in Nov 2015.

However, in that case the DCS will only have a PDF that explains why the data is not shared. For that we don't really need a DUA, or the DUA could be empty.

@hurngchunlee do you have an idea how we could most easily deal with this?

**Comment by hurngchunlee** [comment]
As long as the DSC is not to be closed, the DUA can remain empty.  However, it's the mandatory attribute for closing DSC.  Shall we allow exceptions?

**Comment by robertoostenveld** [comment]
Or should we add a DUA "Not applicable", i.e. a DUA text (and label) that can be attached in case it does not apply? That would allow for not making any exceptions in the IT implementation.

However, the DUA also has consequences for sharing, i.e. at the moment the (remote) researcher wants to access the data, he first has to click through the DUA. So the "N/A" DUA text (read by the remote researcher) should not be empty, it should state that there are no restrictions.   

In short, the DUA would be called "Not applicable" and would contain the text "No restriction apply". The person requesting access would click "OK" and then see the document with the approval that there is no data. 

Would that work?

**Comment by robertoostenveld** [comment]
Rather than "not applicable", we should use the https://en.wikipedia.org/wiki/Public_domain

**Comment by robertoostenveld** [comment]
[master 7b9d3c0] FIX #22 - added public domain as option for the data use agreement.
 3 files changed, 7 insertions(+)
 create mode 100644 dua/PD.html
 create mode 100644 dua/PD.txt
